ykcompile/src/lib.rs

```diff
@@ -45,28 +45,28 @@ lazy_static! {
 
 macro_rules! binop_add_sub {
     ($name: ident, $op:expr) => {
-        fn $name(&mut self, dest: &IPlace, size: u64, opnd2: &IPlace) {
+        fn $name(&mut self, dest: &IPlace, size: u64, opnd2: &IPlace, temp_reg: u8) {
```

Comment:
It seems odd that `dest` is only used for its type.

I have an idea for making this a bit more sane, but I can try in a fresh PR. We really should move on with this one :P